src/wm_info_provider/hyprland.rs

```diff
@@ -52,7 +52,7 @@ impl WmInfoProvider for HyprlandInfoProvider {
     fn get_tags(&self, output: &Output) -> Vec<Tag> {
         self.workspaces
             .iter()
-            .filter(|ws| ws.monitor == output.name)
+            .filter(|ws| ws.monitor == output.name && ws.id >= 0)
```

Comment:
```suggestion
            .filter(|ws| ws.monitor == output.name && !ws.name.starts_with("special:"))
```
(One minor issue: Hyprland allows the names of named workspaces to start with `special:` so these will still be filtered out. However, Hyprland does not expose the information whether a workspace is special in any other manner, so I don't think there is an alternative here)